docs/why-logfire/opentelemetry.md

```diff
@@ -0,0 +1,53 @@
+# OpenTelemetry under the hood :telescope:
+
+Because **Pydantic Logfire** is built on [OpenTelemetry](https://opentelemetry.io/), you can
+use a wealth of existing tooling and infrastructure, including
+[instrumentation for many common Python packages](https://opentelemetry-python-contrib.readthedocs.io/en/latest/index.html). Logfire also supports cross-language data integration and data export to any OpenTelemetry-compatible backend or proxy.
+
+For example, we can instrument a simple FastAPI app with just 2 lines of code:
+
+```py title="fastapi_example.py" hl_lines="8 9 10"
+from datetime import date
+import logfire
+from pydantic import BaseModel
+from fastapi import FastAPI
+
+app = FastAPI()
+
+logfire.configure()
+logfire.instrument_fastapi(app)  # (1)!
+# next, instrument your database connector, http library etc. and add the logging handler (2)
+
+
+class User(BaseModel):
+    name: str
+    country_code: str
+    dob: date
+
+
+@app.post('/')
+async def add_user(user: User):
+    # we would store the user here
+    return {'message': f'{user.name} added'}
+```
+
+1. In addition to [configuring logfire](../reference/configuration.md) this line is all you need to instrument a FastAPI app with Logfire. The same applies to most other popular Python web frameworks.
+2. The [integrations](../integrations/index.md) page has more information on how to instrument other parts of your app.
+
+We'll need the [FastAPI contrib package](../integrations/fastapi.md), FastAPI itself and uvicorn installed to run this:
+
+```bash
+pip install 'logfire[fastapi]' fastapi uvicorn  # (1)!
```

Comment:
Why don't we use the plugin, as it was before? The docs have all installations under the plugin, so it's friendly to our users's setup.

Reply:
I didn't make any changes here, this is the same markdown as it was previously on the all-in-one page, compare this to the current live site. 

![Screenshot 2024-09-25 at 13 15 35](https://github.com/user-attachments/assets/e2a786f0-4603-4800-880d-cdc19dd14b7a)

I'm fine to change this to use the plugin, confirming you mean `{{ install_logfire() }}` ?